lib/vmpooler/api/v1.rb

```diff
@@ -289,6 +293,33 @@ def update_user_metrics(operation, vmname)
       puts 'd', "[!] [#{poolname}] failed while evaluating usage labels on '#{vmname}' with an error: #{e}"
     end
 
+    def reset_pool_size(poolname)
+      result = { 'ok' => false }
+
+      pool_index_live = pool_index(pools)
```

Comment:
Is there anything else you'd like me to do with this bit?

Reply:
if the contents are the same we could use just one variable.

Reply:
I just looked at the api docs and it appears that entire pools cannot be added or removed via it so I think it is safe to make this a single variable. I had done two different ones because I thought you might be able to change the actual list of pools post-startup.

Reply:
Fixed. I am also going to do a different pr to remove all the calls to the pool_index method as the index is created in lib/vmpooler.rb and shouldn't need to be recreated constantly